inference-engine/samples/benchmark_app/README.md

```diff
@@ -95,6 +95,7 @@ Options:
     -layout                     Optional. Prompts how network layouts should be treated by application. For example, "input1[NCHW],input2[NC]" or "[NCHW]" in case of one input size.
     -cache_dir "<path>"         Optional. Enables caching of loaded models to specified directory.
     -load_from_file             Optional. Loads model from file directly without ReadNetwork.
+    -latency_percentile         Optional. Defines the percentile which will be reported in latency metric. The applicable values range is [0, 100]. The default value is 50.
```

Comment:
-latency_percentile         Optional. Defines the percentile to be reported in latency metric. The valid range is [**1**, 100]. The default value is 50 (median).

Reply:
```suggestion
    -latency_percentile         Optional. Defines the percentile to be reported in latency metric. The valid range is [1, 100]. The default value is 50 (median).
```